libxrdp/xrdp_orders_rail.c

```diff
@@ -191,15 +191,18 @@ xrdp_orders_send_as_unicode(struct stream *s, const char *text)
     int str_chars;
     int index;
     int i32;
-    twchar wdst[256];
-
-    str_chars = g_mbstowcs(wdst, text, 255);
+    twchar *wdst;
 
+    wdst = (twchar *) g_malloc(sizeof(twchar) * 64 * 1024, 1);
```

Comment:
Please use `g_new0`, it's more readable.

Why is 64*1024 used? I believe `strlen(text) + 1` should be safe. It's suggested by the glibc documentation for `mbstowcs`: https://www.gnu.org/software/libc/manual/html_node/Non_002dreentrant-String-Conversion.html